### PR TITLE
Slash clothes prices in half or more.

### PR DESCRIPTION
**Pull request by lbnesquik**
Cheaper clothes, more fun. Why are T shirts and gloves 200 thalers anyway?

This is the same PR as #8147 , i just messed up with my git and i pushed to master, forcibly closing that one PR.

### CODE REVIEW

**Comment by lbnesquik** [comment]
> 
> 
> Sorry I have not responded to this sooner, but, the main reason for halving the prices instead of just reducing everything to 50 is because the current prices are based off of loadout prices.

I just don't think those prices are really based on anything sensible but i'll do it

File: code/game/machinery/vending_vr.dm [hunk 1]
```diff
@@ -1205,7 +1205,7 @@
 					/obj/item/clothing/mask/gas/plaguedoctor/fluff = 600,
 					/obj/item/clothing/under/owl = 400,
 					/obj/item/clothing/mask/gas/owl_mask = 400,
-					/obj/item/clothing/under/waiter = 200,
+					/obj/item/clothing/under/waiter = 50,
```
Comment (Novacat) [inline code/game/machinery/vending_vr.dm:1208]: Should be 100

File: code/game/machinery/vending_vr.dm [hunk 2]
```diff
@@ -1216,7 +1216,7 @@
 					/obj/item/clothing/suit/imperium_monk = 2000,
 					/obj/item/clothing/suit/holidaypriest = 200,
 					/obj/item/clothing/head/witchwig = 200,
-					/obj/item/clothing/under/sundress = 200,
+					/obj/item/clothing/under/sundress = 50,
```
Comment (Novacat) [inline code/game/machinery/vending_vr.dm:1219]: Same here.

File: code/game/machinery/vending_vr.dm [hunk 3]
```diff
@@ -389,116 +389,116 @@
...
+					/obj/item/clothing/glasses/monocle = 50,
+					/obj/item/clothing/glasses/goggles = 50,
+					/obj/item/clothing/glasses/fluff/spiffygogs = 50,
+					/obj/item/clothing/glasses/fakesunglasses = 50,
+					/obj/item/clothing/glasses/fakesunglasses/aviator = 50,
+					/obj/item/clothing/mask/bandana/blue = 50,
+					/obj/item/clothing/mask/bandana/gold = 50,
+					/obj/item/clothing/mask/bandana/green = 50,
+					/obj/item/clothing/mask/bandana/red = 50,
+					/obj/item/clothing/mask/surgical = 50)
```
Comment (Novacat) [inline code/game/machinery/vending_vr.dm:501]: This should be 100 as well

File: code/game/machinery/vending_vr.dm [hunk 4]
```diff
@@ -182,64 +182,64 @@
...
+					/obj/item/clothing/gloves/grey = 50,
+					/obj/item/clothing/gloves/sterile/latex = 50,
+					/obj/item/clothing/gloves/light_brown = 50,
+					/obj/item/clothing/gloves/sterile/nitrile = 50,
+					/obj/item/clothing/gloves/orange = 50,
+					/obj/item/clothing/gloves/purple = 50,
+					/obj/item/clothing/gloves/red = 50,
+					/obj/item/clothing/gloves/fluff/siren = 50,
+					/obj/item/clothing/gloves/white = 50,
+					/obj/item/clothing/gloves/duty = 50,
```
Comment (Novacat) [inline code/game/machinery/vending_vr.dm:201]: Gloves should be 100 each

File: code/game/machinery/vending_vr.dm (hunk 1 above)
Comment (lbnesquik) [reply code/game/machinery/vending_vr.dm:1208]: Uniform item, so i think it should be 50 IMO

File: code/game/machinery/vending_vr.dm (hunk 2 above)
Comment (lbnesquik) [reply code/game/machinery/vending_vr.dm:1219]: Uniform item, so i think it should be 50 IMO

**Comment by lbnesquik** [comment]
Waiting on a polaris merge

**Comment by KasparoVy** [comment]
They're gucci gloves!

File: code/game/machinery/vending_vr.dm (hunk 4 above)
Comment (lbnesquik) [reply code/game/machinery/vending_vr.dm:201]: Gloves PR has been merged at polaris
https://github.com/PolarisSS13/Polaris/pull/7287

File: code/game/machinery/vending_vr.dm (hunk 3 above)
Comment (lbnesquik) [reply code/game/machinery/vending_vr.dm:501]: image
Bandanas are 1 point tho.

**Comment by lbnesquik** [comment]
I think this should be ready for merge. @Novacat

File: code/game/machinery/vending_vr.dm [hunk 5]
```diff
@@ -182,64 +182,64 @@
...
-					/obj/item/clothing/shoes/footwraps = 100)
+	prices = list(/obj/item/clothing/gloves/evening = 50,
+					/obj/item/clothing/gloves/fingerless = 50,
+					/obj/item/clothing/gloves/black = 50,
+					/obj/item/clothing/gloves/blue = 50,
+					/obj/item/clothing/gloves/brown = 50,
+					/obj/item/clothing/gloves/color = 50,
+					/obj/item/clothing/gloves/green = 50,
+					/obj/item/clothing/gloves/grey = 50,
+					/obj/item/clothing/gloves/sterile/latex = 50,
```
Comment (Novacat) [inline code/game/machinery/vending_vr.dm:193]: 100

File: code/game/machinery/vending_vr.dm [hunk 6]
```diff
@@ -182,64 +182,64 @@
...
+					/obj/item/clothing/gloves/light_brown = 50,
+					/obj/item/clothing/gloves/sterile/nitrile = 50,
```
Comment (Novacat) [inline code/game/machinery/vending_vr.dm:195]: 100

File: code/game/machinery/vending_vr.dm [hunk 7]
```diff
@@ -182,64 +182,64 @@
...
+					/obj/item/clothing/shoes/hitops/black = 50,
+					/obj/item/clothing/shoes/hitops/blue = 50,
+					/obj/item/clothing/shoes/hitops/green = 50,
+					/obj/item/clothing/shoes/hitops/orange = 50,
+					/obj/item/clothing/shoes/hitops/purple = 50,
+					/obj/item/clothing/shoes/hitops/red = 50,
+					/obj/item/clothing/shoes/flats/white/color = 50,
+					/obj/item/clothing/shoes/hitops/yellow = 50,
+					/obj/item/clothing/shoes/boots/jackboots = 50,
+					/obj/item/clothing/shoes/boots/jungle = 50,
```
Comment (Novacat) [inline code/game/machinery/vending_vr.dm:220]: 100

File: code/game/machinery/vending_vr.dm [hunk 8]
```diff
@@ -182,64 +182,64 @@
...
+					/obj/item/clothing/gloves/red = 50,
+					/obj/item/clothing/gloves/fluff/siren = 50,
+					/obj/item/clothing/gloves/white = 50,
+					/obj/item/clothing/gloves/duty = 50,
+					/obj/item/clothing/shoes/athletic = 50,
+					/obj/item/clothing/shoes/boots/fluff/siren = 50,
+					/obj/item/clothing/shoes/slippers = 50,
+					/obj/item/clothing/shoes/boots/cowboy/classic = 50,
+					/obj/item/clothing/shoes/boots/cowboy = 50,
+					/obj/item/clothing/shoes/boots/duty = 50,
```
Comment (Novacat) [inline code/game/machinery/vending_vr.dm:207]: 100

File: code/game/machinery/vending_vr.dm (hunk 1 above)
Comment (Novacat) [inline code/game/machinery/vending_vr.dm:1208]: 100

File: code/game/machinery/vending_vr.dm (hunk 2 above)
Comment (Novacat) [inline code/game/machinery/vending_vr.dm:1219]: 100

File: code/game/machinery/vending_vr.dm (hunk 4 above)
Comment (Novacat) [inline code/game/machinery/vending_vr.dm:201]: gloves/duty should be 150

**Pull request merged**